tss-esapi/src/structures/tagged_pcr_select.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    constants::PcrPropertyTag,
+    structures::{PcrSelectSize, PcrSlot},
+    tss2_esys::{TPM2_PCR_SELECT_MAX, TPMS_TAGGED_PCR_SELECT},
+    Error, Result, WrapperErrorKind,
+};
+use enumflags2::BitFlags;
+use log::error;
+use std::convert::TryFrom;
+use std::iter::FromIterator;
+/// Type that holds information regarding
```

Comment:
This is a minor nit but not having a newline before doc-comment catches my eye :)

Reply:
Me to. Fixed it now.